include/url.h

```diff
@@ -79,6 +79,11 @@ namespace Url
          */
         Url& abspath();
 
+        /**
+         * Evaluate 'other' relative to this URL, placing the result in this object.
+         */
+        Url& relative(const std::string& other);
```

Comment:
Yes, we definitely can. I think I'll do so in the `url-py` bindings to keep this API clean.